jetty-alpn/jetty-alpn-conscrypt-server/src/main/java/org/eclipse/jetty/alpn/conscrypt/server/ConscryptServerALPNProcessor.java

```diff
@@ -60,9 +62,10 @@ public void configure(SSLEngine sslEngine,Connection connection)
     {
         try
         {
-            Method method = sslEngine.getClass().getMethod("setHandshakeApplicationProtocolSelector", BiFunction.class);
-            method.setAccessible(true);
-            method.invoke(sslEngine,new ALPNCallback((ALPNServerConnection)connection));
+            // with java 9+ we must use
+            //sslEngine.setHandshakeApplicationProtocolSelector(new ALPNCallback((ALPNServerConnection)connection));
+            Conscrypt.setApplicationProtocolSelector(sslEngine,
+                                                      toApplicationProtocolSelector(new ALPNCallback((ALPNServerConnection)connection)));
```

Comment:
The comment seems off here, as in 9.4.x we cannot use Java 9+ APIs.

Since `ALPNCallback` is a private inner class, why don't you make it extend `ApplicationProtocolSelector` rather than wrapping it?